CONTRIBUTING.md

```diff
@@ -180,6 +181,22 @@ base](https://github.com/mlflow/mlflow/blob/cdc6a651d5af0f29bd448d2c87a198cf5d32
 For more information about Plugins, see
 <https://mlflow.org/docs/latest/plugins.html>.
 
+### Python Style Guide
+##### Docstrings
+We follow [Google's Python Style Guide](https://google.github.io/styleguide/pyguide.html) for writing 
+docstrings. Make sure your docstrings adhere to this style guide.
+
+Currently, the MLflow team is working to reformat docstrings to 
+adhere to these guidelines. Any contributions regarding style 
+reformatting are much appreciated!
```

Comment:
The process is ongoing for converting to a standard docstring format style. If you see a docstring in the code base that doesn't adhere to this formatting style and you'd like to contribute a fix, feel free to open a PR to correct the docstring formatting.